raiden-dapp/tests/unit/components/transfer/transfer-menus.spec.ts

```diff
@@ -1,5 +1,10 @@
+jest.mock('vue-router');
+import Mocked = jest.Mocked;
 import { mount, Wrapper } from '@vue/test-utils';
 import Vue from 'vue';
+import store from '@/store';
```

Comment:
The store is not used in these test. Please remove this here.

Reply:
It is, not including the store will give you this error when running the tests: `Cannot read property 'getters' of undefined`.

Reply:
Okay so this means that the component requires some state right? Do you know what this data looks like? Is it okay that the store is empty? I can't see that. So I would mock it in the unit test. Else this is out of control and nobody knows where this eventually fails when we change the store.

Reply:
As I understand it if the store is used somewhere in the component being tested it also needs to be imported in the test wrapper setup. At least this is how I learned to do it from previous developers.